tests/ui_tools/test_boxes.py

```diff
@@ -83,9 +83,12 @@ def test_not_calling_typing_method_without_recipients(self, mocker, write_box):
             ("#**Stream 1>", 1, "#**Stream 1>This is a topic**", True),
             ("#**Stream 1>", -1, "#**Stream 1>Hello there!**", True),
             ("#**Stream 1>", -2, "#**Stream 1>This is a topic**", True),
+            # Fenced prefix
+            ("#**Stream 1**>T", 0, "#**Stream 1>Topic 1**", True),
```

Comment:
For reference, the web app also supports `#**Stream 1** >`, presumably for the stream complete generating a space after it?

I think this could be an extra feature on top, depending how straightforward it is, since we don't add a space.

Reply:
Yeah. I just quickly tested some autocompletes in the web app. All of them added a space in the end. For us, that might be a future normal, I think :)